test/Spec.hs

```diff
@@ -71,6 +71,12 @@ main = hspec .
            |3 | import Prelude (tail, head, Integer, (+))
            |  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^|]]
 
+    expectWarnings
+      "test/UnorderedMultipleDeriving.hs"
+      [[str|Unsorted multiple deriving expected: deriving newtype Eq, deriving stock Show
```

Comment:
@mbj Looking at this (and a line 69 in this file), it reads "weird" because there's no comma or dot before `expected`. The way this reads is basically saying that the violation is expected.

Reply:
@snusnu lets go for `expected order` than.

Reply:
@mbj The key to me seems to be to separate the `expected` state from the "violation name" via some sort of separator character.